public/app/features/alerting/unified/components/RuleLocation.tsx

```diff
@@ -9,13 +9,17 @@ interface Props {
 
 const RuleLocation = ({ namespace, group }: Props) => {
   if (!group) {
-    return <>{namespace}</>;
+    return (
+      <Text element="h3" variant="body">
+        {namespace}
+      </Text>
+    );
   }
 
   return (
-    <>
-      {namespace} <Icon name="angle-right" /> {group}
-    </>
+    <Text element="h3" variant="body">
+      {namespace} <Icon name="angle-right" aria-label=">" /> {group}
```

Comment:
Wouldn't this read a bit weird for screen readers? What if we labeled it as "with group" or similar?

Reply:
At the moment  the `>` isn't being read at all, but without it there is no space, weirdly enough. I like the `with group` idea, although I would probably want to make in an i18n string altoghether.